WordPress/src/main/java/org/wordpress/android/ui/posts/PostEditorAnalyticsSession.java

```diff
@@ -76,10 +78,12 @@
         }
     }
 
-    public void start(boolean hasUnsupportedBlocks) {
+    public void start(ArrayList<Object> unsupportedBlocksList) {
         if (!mStarted) {
-            mHasUnsupportedBlocks = hasUnsupportedBlocks;
             Map<String, Object> properties = getCommonProperties();
+            mHasUnsupportedBlocks = unsupportedBlocksList != null && unsupportedBlocksList.size() > 0;
```

Comment:
Hmm, actually, we need to compute `mHasUnsupportedBlocks` before the getCommonProperties() call because it's used there.

Reply:
Oh right, fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/10237/commits/bd5282fa353eedc968306166a1fb814c4f7f39d9